src/Service/RoutingLoader/RoutingAttributeLoader.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Paysera\Bundle\ApiBundle\Service\RoutingLoader;
+
+use Paysera\Bundle\ApiBundle\Annotation\RestAnnotationInterface;
+use Paysera\Bundle\ApiBundle\Attribute\RestAttributeInterface;
+use Paysera\Bundle\ApiBundle\Service\RestRequestHelper;
+use ReflectionClass;
+use ReflectionMethod;
+use Symfony\Bundle\FrameworkBundle\Routing\AttributeRouteControllerLoader;
+use Symfony\Component\Routing\Route;
+
+/**
+ * @internal
+ */
+class RoutingAttributeLoader extends AttributeRouteControllerLoader
+{
+    private const ATTRIBUTE_SUPPORT_PHP_VERSION_ID = 80100; // `new` in initializers support is required
+
+    /**
+     * @var RestRequestHelper
+     */
+    private $restRequestHelper;
+
+    /**
+     * @var RestRequestAnnotationOptionsBuilder
+     */
+    private $annotationOptionsBuilder;
+
+    /**
+     * @var RestRequestAttributeOptionsBuilder
+     */
+    private $attributeOptionsBuilder;
+
+    public function setRequestHelper(RestRequestHelper $restRequestHelper)
+    {
+        $this->restRequestHelper = $restRequestHelper;
+    }
+
+    public function setRestRequestAnnotationOptionsBuilder(RestRequestAnnotationOptionsBuilder $annotationOptionsBuilder
+    ) {
+        $this->annotationOptionsBuilder = $annotationOptionsBuilder;
+    }
+
+    public function setRestRequestAttributeOptionsBuilder(RestRequestAttributeOptionsBuilder $attributeOptionsBuilder)
+    {
+        $this->attributeOptionsBuilder = $attributeOptionsBuilder;
+    }
+
+    protected function configureRoute(
+        Route $route,
+        ReflectionClass $class,
+        ReflectionMethod $method,
+        object $annot
```

Comment:
It appears that PHP 7.2 [introduces the `object` type-hint](https://www.php.net/releases/7_2_0.php).